src/spec/book_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative '../classes/book'
+
+describe Book do
+  before(:each) do
+    @book = Book.new('publisher', 'good', 1, Date.new(2001, 2, 3), false)
+  end
+
+  context '#new' do
+    it 'is an instance of Book and takes 5 parameter' do
+      expect(@book).to be_instance_of(Book)
+    end
+
+    it 'gives Argument error when more or less than 5 params given' do
+      expect { Book.new('publisher') }.to raise_error(ArgumentError)
+    end
+
+    it 'is a child of Item' do
+      expect(@book).to be_kind_of(Item)
+    end
+  end
+
+  context '#publisher' do
+    it 'can be read' do
+      expect(@book.publisher).to eql('publisher')
+    end
+
+    it 'can not be written' do
+      expect { @book.publisher = 3 }.to raise_error(NoMethodError)
+    end
+  end
+
+  context '#cover_state' do
+    it 'can be read' do
+      expect(@book.cover_state).to eql('good')
+    end
+
+    it 'can not be written' do
+      expect { @book.cover_state = 3 }.to raise_error(NoMethodError)
+    end
+  end
+
+  context '#move_to_archive' do
+    it "should return true if cover_state equals 'bad', even if parent's method returns false" do
+      bad_book = Book.new('publisher', 'bad', 1, Date.new(20_015, 2, 3), false)
```

Comment:
The year parameter seems a bit odd. I think it would be better to remove the extra 0 and the underscore